src/connection.ts

```diff
@@ -658,6 +665,20 @@ export class Connection extends EventEmitter {
       }
     }
 
+    // Allow consumer to choose to fulfill each packet and/or perform other logic before fulfilling
```

Comment:
should the [description of `handlePrepare`](https://github.com/interledgerjs/ilp-protocol-stream/blob/5e2c963462937547a8499978e8ad3d41b8baaff1/src/connection.ts#L493-L495) be updated?